.storybook/preview.js

```diff
@@ -12,9 +12,10 @@ export const parameters = {
     storySort: {
       order: [
         `UIG-${version}`,
-        ["Introduction", "Changelog", "Contribute"],
-        "custom-elements",
-        "native-elements",
+        ['Introduction', 'Changelog', 'Contribute'],
+        'custom-elements',
+        'native-elements',
+        'legacy-elements',
```

Comment:
Ik zou `legacy-elements` vervangen door gewoon `legacy`. Dan komt het overeen met de folder, en is het gewoon een 'legacy' hoofdstuk binnen onze repo.